src/components/ResultsMap.vue

```diff
@@ -29,11 +34,12 @@ export default {
     return {
 			markers: this.searchResults,
 			centerMap: {
-				lat: 40, lng: -105.0465427
+				lat: this.$store.state.geolocation.coords.latitude, 
+				lng: this.$store.state.geolocation.coords.longitude
```

Comment:
Thanks for getting these hard-coded values out of the file.  It'll be nice for the user to see the map centered at their current location.